Makefile

```diff
@@ -3,8 +3,27 @@ runtime-upgrade-test:
 	cargo build -p $(runtime)-runtime --release --locked
 	cd tests/e2e && yarn --frozen-lockfile && yarn test:runtime-upgrade-$(runtime)
 
+# use `cargo nextest run` if cargo-nextest is installed, fallback cargo test
+cargo_test = $(shell which cargo-nextest >/dev/null && echo "cargo nextest run" || echo "cargo test")
+
+.PHONY: test
+test:
+	SKIP_WASM_BUILD= ${cargo_test} --all
```

Comment:
I think we want to skip `--all` since it disables some part of the code (even though I think they're not tested by UTs) that are run outside of `evm-tracing` context. This is why we specified `try-runtime,runtime-benchmarks` explicitly before.

Reply:
actually `--all` is deprecated and needs to be replaced with `--workspace`. I don't understand `it disables some part of the code`

Reply:
Some parts of the client code are compiled only when `evm-tracing` is disabled, and vice-versa. That's what I meant by disabled - poor choice of words from my side.

Since non-evm-tracing client is _more important_, running tests with disabled `evm-tracing` makes more sense. But then again, I don't believe we have tests covering client so perhaps this worry is baseless.

Reply:
We don't run any test with feature `evm-tracing`. Will add one